Makefile

```diff
@@ -5,9 +5,10 @@ all: directory
 
 directory:
 	mkdir directory
-	export URL=$$( curl --silent "https://api.github.com/repos/c4dt/lightarti-directory/releases/latest" | \
-		jq -r '.assets[0].browser_download_url' ); \
-	wget -q -O- $$URL | tar -C directory -xz
+	curl --silent "https://api.github.com/repos/c4dt/lightarti-directory/releases/latest" | \
```

Comment:
Why not download directly from

https://github.com/c4dt/lightarti-directory/releases/latest/download/directory-cache.tgz

?

Reply:
I used the method already present in the Makefile but yeah, using the direct URL is way easier